static/backbone/js/api/lib/maps/overlays/infobubbles/base.js

```diff
@@ -96,27 +98,43 @@ define(['jquery',
             },
 
             renderViewContent: function () {
-                var template = this.getTemplate("InfoBubbleTemplate");
+                var template = this.getTemplate("InfoBubbleTemplate"),
+                    that = this;
                 this.$el = $(template(this.getContext()));
-
+                this.$el.click(function (e) {
+                    that.bringToFront(e);
+                });
                 this.showUpdatedContent();
             },
-
+            bringToFront: function (e) {
+                var zIndex;
+                //console.log(this.bubble.bubble_.style.zIndex);
+                zIndex = parseInt(this.bubble.bubble_.style.zIndex, 10);
+                this.bubble.bubble_.style.zIndex = zIndex + 1;
+                e.preventDefault();
```

Comment:
What's the intended behavior here?  Every time you click the bubble its z-index goes up by 1, unbounded.  Is that intentional?